locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -612,6 +612,9 @@ public MachineLocation obtain(Map<?,?> flags) throws NoMachinesAvailableExceptio
         for (int i = 1; i <= attempts; i++) {
             try {
                 return obtainOnce(setup);
+            } catch (IllegalArgumentException e) {
+                LOG.warn("Attempt #{}/{} to obtain machine threw error: {}", new Object[]{i, attempts, e});
+                exceptions.add(e);
```

Comment:
Same as above, the catch block is unnecessary as `RuntimeException` is already caught below